CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 10.1.0
```

Comment:
Some other changes have merged recently so this file now conflicts. And since this change isn't currently tied to a version release you should add your update under `## HEAD` when you resolve the conflict.